tests/node_tests/node_tests.exp

```diff
@@ -1565,7 +1565,7 @@ Error --------------------------------------------------------------------------
 Unexpected end of input, expected a valid JSON value
 
    1|
-      
```

Comment:
removing trailing whitespace is why the tests are failing 😞

try `runtest.sh -r` again, I guess? unless it's git that's stripping it?

Reply:
Nah it was me. Crazy didn't expect that but will keep it in mind next time